haystack/pipelines/ray.py

```diff
@@ -80,14 +83,15 @@ def load_from_config(
         strict_version_check: bool = False,
         address: Optional[str] = None,
         ray_args: Optional[Dict[str, Any]] = None,
+        serve_detached: Optional[bool] = False,
```

Comment:
Hi @zoltan-fedor I think we are reserving the use of `Optional` for when the default value of the option is `None`. So I think in this case we should just use `bool` as the type for `serve_detached`.

Reply:
Apologies, an oversight, this has been fixed now.